sampleconfig/configtx.yaml

```diff
@@ -141,19 +141,22 @@ Application: &ApplicationDefaults
         # (e.g.,who can receive Block events). This section does NOT specify the resource's
         # definition or API, but just the ACL policy for it.
         #
-        # User's can override these defaults with their own policy mapping by defining the
+        # Users can override these defaults with their own policy mapping by defining the
         # mapping under ACLs in their channel definition
 
         #---New Lifecycle System Chaincode (_lifecycle) function to policy mapping for access control--#
 
+        # ACL policy for _lifecycle's "CheckCommitReadiness" function
+        _lifecycle/CheckCommitReadiness: /Channel/Application/Writers
+
         # ACL policy for _lifecycle's "CommitChaincodeDefinition" function
         _lifecycle/CommitChaincodeDefinition: /Channel/Application/Writers
 
         # ACL policy for _lifecycle's "QueryChaincodeDefinition" function
         _lifecycle/QueryChaincodeDefinition: /Channel/Application/Readers
 
```

Comment:
Are these two different?

       # ACL policy for _lifecycle's "QueryChaincodeDefinition" function
        _lifecycle/QueryChaincodeDefinition: /Channel/Application/Readers

        # ACL policy for _lifecycle's "QueryChaincodeDefinitions" function
        _lifecycle/QueryChaincodeDefinitions: /Channel/Application/Readers

Reply:
Yes, they are two different queries.